### ISSUE TITLE
Jade 11.9.1 Fabric Crash on Startup

### ISSUE THREAD

**Issue by ninjaeon** [Snownee/Jade#402]
### Mod loader

Fabric

### Minecraft version

1.20.1

### Mod version

11.9.1

### Modloader version

Fabric: Loader 0.15.10 + API 0.92.1

### If bug:

- [ ] Can you reproduce this issue with relevant mods only?

### If bug: The latest.log file

https://mclo.gs/TASn5nf

### Issue description

Jade 11.9.1 is crashing on startup for me.  Reverting back to Jade-1.20-fabric-11.9.0 resolves the issue.

Possible relevant lines in attached log:
````
[03:17:38] [Render thread/ERROR]: Error loading plugin at snownee.jade.addon.vanilla.VanillaPlugin
java.lang.IllegalArgumentException: Duplicate config key: minecraft:harvest_tool.effective_tool
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:220) ~[guava-31.1-jre.jar:?]
	at snownee.jade.impl.config.PluginConfig.addConfig(PluginConfig.java:50) ~[Jade-1.20-fabric-11.9.1.jar:?]
	at snownee.jade.impl.WailaClientRegistration.addConfig(WailaClientRegistration.java:233) ~[Jade-1.20-fabric-11.9.1.jar:?]
	at snownee.jade.addon.vanilla.VanillaPlugin.registerClient(VanillaPlugin.java:130) ~[Jade-1.20-fabric-11.9.1.jar:?]
	at snownee.jade.util.CommonProxy.lambda$loadComplete$5(CommonProxy.java:381) ~[Jade-1.20-fabric-11.9.1.jar:?]
	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?]
	at snownee.jade.util.CommonProxy.loadComplete(CommonProxy.java:368) ~[Jade-1.20-fabric-11.9.1.jar:?]
	at snownee.jade.util.ClientProxy.lambda$onInitializeClient$1(ClientProxy.java:247) ~[Jade-1.20-fabric-11.9.1.jar:?]
	at net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientLifecycleEvents.lambda$static$0(ClientLifecycleEvents.java:35) ~[fabric-lifecycle-events-v1-2.2.2 1802ada577-64bb13225b65065c.jar:?]
	at net.minecraft.client.MinecraftClient.handler$fia000$fabric-lifecycle-events-v1$onStart(MinecraftClient.java:20033) ~[client-intermediary.jar:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:781) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:250) ~[minecraft-1.20.1-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.10.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.10.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.10.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:87) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:130) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) ~[NewLaunch.jar:?]
[03:17:38] [Render thread/ERROR]: Unhandled game exception
java.lang.IllegalArgumentException: Duplicate config key: minecraft:harvest_tool.effective_tool
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:220) ~[guava-31.1-jre.jar:?]
	at snownee.jade.impl.config.PluginConfig.addConfig(PluginConfig.java:50) ~[Jade-1.20-fabric-11.9.1.jar:?]
	at snownee.jade.impl.WailaClientRegistration.addConfig(WailaClientRegistration.java:233) ~[Jade-1.20-fabric-11.9.1.jar:?]
	at snownee.jade.addon.vanilla.VanillaPlugin.registerClient(VanillaPlugin.java:130) ~[Jade-1.20-fabric-11.9.1.jar:?]
	at snownee.jade.util.CommonProxy.lambda$loadComplete$5(CommonProxy.java:381) ~[Jade-1.20-fabric-11.9.1.jar:?]
	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?]
	at snownee.jade.util.CommonProxy.loadComplete(CommonProxy.java:368) ~[Jade-1.20-fabric-11.9.1.jar:?]
	at snownee.jade.util.ClientProxy.lambda$onInitializeClient$1(ClientProxy.java:247) ~[Jade-1.20-fabric-11.9.1.jar:?]
	at net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientLifecycleEvents.lambda$static$0(ClientLifecycleEvents.java:35) ~[fabric-lifecycle-events-v1-2.2.2 1802ada577-64bb13225b65065c.jar:?]
	at net.minecraft.client.MinecraftClient.handler$fia000$fabric-lifecycle-events-v1$onStart(MinecraftClient.java:20033) ~[client-intermediary.jar:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:781) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:250) ~[minecraft-1.20.1-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.10.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.10.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.10.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:87) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:130) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) ~[NewLaunch.jar:?]
````

**Comment by Snownee** [comment]
it's a bug in Infinity Buttons. before it is fixed, please downgrade Jade to 11.9.0

**Comment by Snownee** [comment]
fixed now
